bibliopixel/drivers/curses.py

```diff
@@ -0,0 +1,91 @@
+import curses as _curses
```

Comment:
You need to wrap this up so it does something sane on windows because python does not include this module on Windows. Please make it fail gracefully.